src/api-v1.js

```diff
@@ -103,10 +106,17 @@ api.declare({
             url: url,
           });
         }
+        // TODO: Should only status === 'error' be considered a miss?  A file
+        // that's never been requested before could never be in the cache, so
+        // it not being there really is not an error case
+        this.monitor.count(`${service}.${region}.cache-miss`, 1);
```

Comment:
Also I'm tempted to suggest that you record two metrics for each of these. Specifically:
```js
this.monitor.count(`${service}.${region}.cache-miss', 1);
this.monitor.count(`cache-miss`, 1);
```

------

@imbstack, I don't think neither syntax here is supported:
```js
this.monitor.prefix({service, region}).count('cache-miss', 1);
this.monitor.count([{service, region}, 'cache-miss'], 1);
```
Statsum does support: `statsum.count({service, region}, 1);`
But I don't think monitor does... The result would be:
```
service:<service>.region:<region>
service:<service>.all-region
all-service.region:<region>
all-service.all-region
```
It does create an awful high dimensional, so I'm not even sure it's a good idea...
Maybe we should remove the pattern from node-statsum? Rather than adding it to tc-lib-monitor.
If we did, add it we would probably want to add the extension: `this.monitor.count([{service, region}, 'cache-miss'], 1);`, as prefix() probably should only be a fixed string, and we would need to append the `cache-miss` bit.

@jhford sorry, this totally got off track from this review... Only related in the sense that maybe it could have been smart... most likely not a good feature anyways...